website/data/docs-remote-plugins.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "title": "Scaffolding",
+    "path": "scaffolding",
+    "repo": "zchsh/packer-plugin-scaffolding"
+  },
+  {
+    "title": "Docker",
+    "path": "docker-remote-plugin",
+    "repo": "zchsh/packer-plugin-docker"
+  }
```

Comment:
I've left some entries here for demo purposes. I'll remove these once we're happy with this approach and feel this PR is ready to be merged. Then, they can be added back into this file in the PRs that move the plugin source out of `packer`, eg #10695 .

- [ ] @zchsh  Change this file to be an empty array before merging

Reply:
Pretty soon we can replace this with an an official example https://github.com/hashicorp/packer-plugin-docker/pull/1